cast/src/lib.rs

```diff
@@ -666,6 +666,7 @@ where
         &self,
         tx_hash: String,
         field: Option<String>,
+        raw: bool,
```

Comment:
Is it better to try and rather use the field and do a match on the string as original commit? Current way seems cleaner, but I am worried about a breaking change in the package.

Reply:
this is fine